client/src/modules/general-ledger/general-ledger.html

```diff
@@ -65,6 +65,14 @@
           ng-class="{ 'btn-info' : GeneralLedgerCtrl.filterEnabled }">
           <i class="fa fa-filter"></i>
         </button>
+        <button
+          type="button"
+          ng-click="GeneralLedgerCtrl.toggleHideTitlesAccounts()"
+          data-method="hide_titles"
+          class="btn btn-default"
+          ng-class="{ 'btn-info' : GeneralLedgerCtrl.hideTitlesAccounts }">
+          <i class="fa fa-eye-slash"></i>
```

Comment:
It might be nice to have the `ng-class` here toggle between `fa-eye` (if title accounts are visible) and `fa-eye-slash` (if title accounts are hidden).